frappe/public/js/frappe/ui/toolbar/search_utils.js

```diff
@@ -87,8 +87,8 @@ frappe.search.utils = {
                     if(level) {
                         out.push({
                             type: "In List",
-                            prefix: "Find '" + __(parts[0]).bold() + "' in ",
-                            label: __(me.bolden_match_part(item, parts[1])),
+                            prefix: __("Find '" + __(parts[0]).bold() + "' in "),
+                            label: __(me.bolden_match_part(__(item), parts[1])),
```

Comment:
should also be like `__('Find {0} in {1}')`

try and re-use translation strings